docs/interfaces.md

```diff
@@ -12,184 +12,350 @@ Slots may support multiple connections to plugs.  For example the OS snap
 exposes the ``network`` slot and all applications that can talk over the
 network connect their plugs there.
 
-## Supported Interfaces - Basic
+The availability of an interface depends on whether snapd is running on a
+classic (eg, traditional desktop or server) or on a native system. Interfaces
```

Comment:
The interface set isn't the same though per implicit.go and confirmed by examining snap interfaces on a daily core image and on a desktop system. There are a lot of interfaces that aren't exposed in core: sups-control, gsettings, opengl, pulseaudio, unity7 and x11 and soon content. I wanted to call this out, but I wasn't sure how to do it without going with 'classic vs native'.

Do you have suggestions on how this should be described if not native vs classic? How do I describe those interfaces that only show up on devices that are traditional Linux systems (ie, what I referred to as 'classic').